content/en/docs/concepts/storage/volumes.md

```diff
@@ -194,8 +194,7 @@ keyed with `log_level`.
 
 {{< note >}}
 
-* You must create a [ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/)
-  before you can use it.
+* You must [create a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/#create-a-configmap) before you can use it.
```

Comment:
Would be nice to keep the line break so the line isn't so long. Makes it a little easier to read with editors that don't have line wrapping on. No difference in the rendered output, of course, so just a nit.

Reply:
I have updated the file with the suggested changes.